src/store/item.ts

```diff
@@ -29,12 +29,12 @@ export const useItemStore = defineStore("item", {
       const item: Item = { ...params, id }
 
       this.itemList.push(item)
-      return this.saveData(podcastname, docname)
+      this.saveData(podcastname, docname)
     },
 
     async updateSlotItem(item: [], podcastname: string, docname: string) {
       this.itemList = item
-      return this.saveData(podcastname, docname)
+      this.saveData(podcastname, docname)
```

Comment:
without the return this is fire forget, you've no ability to `await updateSlotItem`

Reply:
I have added `return`